nxc/connection.py

```diff
@@ -22,17 +21,17 @@
 global_failed_logins = 0
 user_failed_logins = {}
 
-
 def gethost_addrinfo(hostname):
-    try:
-        for res in getaddrinfo( hostname, None, AF_INET6, SOCK_DGRAM, IPPROTO_IP, AI_CANONNAME):
-            af, socktype, proto, canonname, sa = res
-        host = canonname if ip_address(sa[0]).is_link_local else sa[0]
-    except socket.gaierror:
-        for res in getaddrinfo( hostname, None, AF_INET, SOCK_DGRAM, IPPROTO_IP, AI_CANONNAME):
-            af, socktype, proto, canonname, sa = res
-        host = sa[0] if sa[0] else canonname
-    return host
+    is_ipv6 = False
+    is_link_local_ipv6 = False
+    for res in getaddrinfo( hostname, None, AF_UNSPEC, SOCK_DGRAM, IPPROTO_IP, AI_CANONNAME):
```

Comment:
need to be tested against windows xp and server 2003

![image](https://github.com/Pennyw0rth/NetExec/assets/5891788/7c82edcb-d360-4c32-b5b3-7b58969c0a6c)

https://stackoverflow.com/questions/8149601/what-would-be-the-disadvantages-risks-of-using-af-unspec

Reply:
Test:
- system: 2003

![image](https://github.com/Pennyw0rth/NetExec/assets/30458572/5f748215-704f-4926-9519-081e982dd148)

Reply:
I have do more `tcpdump` capture test, confirm it will not make any connection to target host

- Provide with IP address
![image](https://github.com/Pennyw0rth/NetExec/assets/30458572/976f8c8e-f71f-47fa-91f3-918d232d58ca)

- Provide with hostname
![image](https://github.com/Pennyw0rth/NetExec/assets/30458572/724fe112-47b7-4633-b29e-e22f0d9d2a5c)